From 1a4d51ec8e2fc0c1a385ea5ff2ec08f129027ef4 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 9 Sep 2016 16:10:10 +0000
Subject: [PATCH 0791/1077] MANTIS 0040175 extension: Components for volume
 control and detecting accessibility mode

platform.idl
V3.4

- added token status: platform token


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1082 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index c2e82ab4..79ba905f 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "3.3";
+    const string interface_version = "3.4";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -51,6 +51,7 @@ module PlatformManagement
         TOKEN_APP_IDLE,   /**< token of a currently AVAILABLE or UNAVAILABLE application */
         TOKEN_MOR,        /**< token of media off rolling module */
         TOKEN_MOR_ACTIVE, /**< token of media off rolling module, that is currently active */
+        TOKEN_PLF,        /**< token of platform */
         TOKEN_INVALID     /**< invalid token */
     };
 
-- 
2.41.0.windows.1

